transport/tchannel/handler.go

```diff
@@ -142,7 +140,7 @@ func (h handler) callHandler(ctx context.Context, call inboundCall) error {
 		return err
 	}
 
-	return h.Handler.Handle(ctx, tchannelOptions, treq, rw)
+	return h.Handler.Handle(ctx, transportOptions, treq, rw)
```

Comment:
So `transportOptions` is where we specify the default options, including requiring enveloping disabled.